crypto/src/mls/session/mod.rs

```diff
@@ -35,15 +35,18 @@ use std::sync::Arc;
 use std::{collections::HashSet, fmt};
 use tls_codec::{Deserialize, Serialize};
 
-/// Represents a MLS client which in our case is the equivalent of a device.
+/// A MLS Session enables a user device to communicate via the MLS protocol.
 ///
-/// It can be the Android, iOS, web or desktop application which the authenticated user is using.
-/// A user has many client, a client has only one user.
-/// A client can belong to many MLS groups
+/// This closely maps to the `Client` term in [RFC 9720], but we avoid that term to avoid ambiguity;
+/// `Client` is very overloaded with distinct meanings.
 ///
-/// It is cheap to clone a `Client` because everything heavy is wrapped inside an [Arc].
+/// There is one `Session` per user per device. A session can contain many MLS groups/conversations.
```

Comment:
> There is one `Session` per user per device.

Hm, but isn't it possible to have, say, 3 different `Session` instances, having their own databases, that all run on the same device and use the same user?

Reply:
I'm abstracting a little. "User" here is authenticated user id; if you have two usernames, you can have additional sessions. "Device" is in the MLS sense of an installation; you can have two Wire apps installed and each is a different device.

That complication didn't seem worth mentioning here though. Do you think it's worth going into that detail?

Reply:
Probably not, but I find the comment confusing because I can imagine, in theory, an app that has one session that uses an on-disk database, and another session that uses in-memory database, with the exact same user and device IDs. The comment implies that would not be possible, but I'm not sure I understand why.